includes/check-local-changes.yml

```diff
@@ -0,0 +1,9 @@
+# requires overwrite_local_changes (bool, dafault to false) to be set
+---
+  - name: Verify local changes
+    shell: "rpm -V obs-api"
+    changed_when: false
+    ignore_errors: false
+    become: true
+    when:
+      - not overwrite_local_changes
```

Comment:
If I understand it correctly, this flag is not preventing overwrites but preventing the execution of the task, right?

Reply:
Correct

Reply:
Maybe we can improve the naming a bit, if this, in fact, is not overwriting the local changes :)

Reply:
What about "skip_local_rpm_check"?

Reply:
Better

Reply:
@guxtavo Do you need some help with this?